cbv/management/commands/load_all_django_versions.py

```diff
@@ -0,0 +1,22 @@
+import os
+import re
+
+from django.conf import settings
+from django.core.management import call_command, BaseCommand
+
+
+class Command(BaseCommand):
+    """Load the Django project fixtures and all version fixtures"""
+
+    def handle(self, **options):
+        fixtures_dir = os.path.join(settings.DIRNAME, 'cbv', 'fixtures')
+        self.stdout.write('Loading project.json')
+        call_command('loaddata', 'cbv/fixtures/project.json')
+        version_fixtures = [re.match(r'((?:\d+\.){2,3}json)', filename) for filename in os.listdir(fixtures_dir)]
```

Comment:
Use [glob](https://docs.python.org/2/library/glob.html#glob.glob) here to save the matching and try/catch block.